os_update-formula/pillar.example

```diff
@@ -8,6 +8,10 @@ os-update:
   # time: 'Thu *-*-01/4 02:00:00'
   time: false
 
+  # additional, optional, overrides for the timer unit
+  accuracysec: 1us
+  randomizeddelaysec: 0
```

Comment:
It would be nicer to use a small value such as `20` to avoid a load spike on the download servers, that could cause new interesting problems (download fails because all workers are busy)

Reply:
I can consider your suggestion for https://gitlab.infra.opensuse.org/infra/salt/-/merge_requests/2054.

Reply:
Cannot hurt to have good examples, too :-)

Reply:
ok ;) added